### PR TITLE
ch(precommitscript): Configure ESLint and pre-commit script

### PR DESCRIPTION
**Pull request by Francoise02**
What does this PR do?
It checks the style violations and flag them.

How should this be manually tested?
Clone this repo, write codes, then hover around the codes. If there's any violation in style ESlint will highlight it. 

Any background context you want to provide?
This ESlint does not consider the typescript

what are the relevant pivotal tracker stories?
[#182201510](https://www.pivotaltracker.com/n/projects/2569104/stories/182201510)

### CODE REVIEW

File: .eslintrc.json
Comment (didasmbalanya) [inline .eslintrc.json]: remove commented code

**Review by didasmbalanya — CHANGES_REQUESTED** [review]
Missing pre commit hook which should format the code and run test

File: .eslintrc.json
Comment (Francoise02) [reply .eslintrc.json]: Sure. I have done git rebase and will remove those commented codes soon. However, the pre-commit that will run the tests is blocked by the setup testing but I'll add it as soon as the setup testing is merged.

**Review by ishimweeli — DISMISSED** [review]

**Comment by Francoise02** [comment]
All ESLint, Prettier, and the pre-commit hook for code formatting function properly. Although, the pre-commit hook for running the tests will be completed after the setup-testing branch is merged to dev.

**Review by Julienish14 — DISMISSED** [review]
great

File: pull_request_template.md
Comment (didasmbalanya) [inline pull_request_template.md]: This is not supposed to be changed

**Review by didasmbalanya — CHANGES_REQUESTED** [review]

**Review by Julienish14 — DISMISSED** [review]

**Review by kalisaNkevin — APPROVED** [review]

**Review by munyanezaarmel — APPROVED** [review]

**Comment by munyanezaarmel** [comment]
Well done @Francoise02

**Review by didasmbalanya — APPROVED** [review]

**Pull request merged**
